python/lsst/ap/verify/ap_verify.py

```diff
@@ -41,7 +43,17 @@
 from .pipeline_driver import ApPipeParser, runApPipeGen2, runApPipeGen3
 from .workspace import WorkspaceGen2, WorkspaceGen3
 
-_LOG = lsst.log.Log.getLogger(__name__)
+_LOG = logging.getLogger(__name__)
+
+
+def _configure_logger():
+    """Configure Python logging.
+
+    Does basic Python logging configuration and
+    forwards LSST logger to Python logging.
+    """
+    logging.basicConfig(level=logging.INFO, stream=sys.stdout)
+    lsst.log.configure_pylog_MDC("DEBUG", MDC_class=None)
```

Comment:
No objection to `INFO`, but what is this last line?

Reply:
Never mind, it's obvious.

Reply:
That forwards all DEBUG and above messages to python logging -- python will drop the actual DEBUG messages. MDC is disabled.